lib/pull.py

```diff
@@ -195,6 +203,7 @@ def fatman_load(self):
          if (arch in self.architectures):
             ch.FATAL("manifest list: duplicate architecture: %s" % arch)
          self.architectures[arch] = ch.digest_trim(digest)
+         self.digests[arch] = digest[7:]
```

Comment:
If the slice is to remove the `sha256:`, I recommend you split on `:`, for greater clarity as well as insulation against the hash algorithm name length changing.